docs/tables/aws_ssm_managed_instance_compliance.md

```diff
@@ -0,0 +1,61 @@
+# Table: aws_ssm_managed_instance_compliance
+
+A managed instance is any machine configured for AWS Systems Manager. You can configure Amazon Elastic Compute Cloud (Amazon EC2) instances or on-premises machines in a hybrid environment as managed instances. Systems Manager supports various distributions of Linux, including Raspberry Pi devices, macOS, and Microsoft Windows Server.
+
+AWS SSM Managed Instance Compliance provide list of compliance statuses for different resource types for a specified resource ID.
+
+**You must specify an Managed Instance ID** in a `where` clause (`where resource_id='`).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  id,
+  name,
+  resource_id,
+  status,
+  compliance_type,
+  severity
+from
+  aws_ssm_managed_instance_compliance
+where
+  resource_id = '2a49c5a11ed9d37f';
```

Comment:
@bigdatasourav In the initial example query results, the resource_id columns were EC2 instance IDs, e.g., `i-abc123456def`. Here the resource IDs look a bit different - what type of resource IDs could a managed instance have, and under which circumstances do we get the different types?

Reply:
@cbruno10 I have provided dummy value in the example , resource id will be instace id only.